src/lib/data/banners/events/3.5.json

```diff
@@ -69,6 +69,71 @@
 					}
 				}
 			}
+		},
+		{
+			"phase": 2,
+			"banners": {
+				"standardVersion": 3,
+				"events": {
+					"item": [
+						{
+							"name": "the-transcendent-one-returns-2",
+							"character": "shenhe", 
+                             "buttonBoxPosition": { 
+ 								"w": 180, 
+ 								"t": 20, 
+ 								"l": 30 
+ 							}, 
+							"textOffset": {
+								"t": 78.5,
+								"l": 43
+							}
+						},
+						{
+							"name": "the-heron_s-court-3",
+						"character": "kamisato-ayaka",
+							"buttonBoxPosition": {
+								"w": 82,
+								"t": 32
+							},
+							"textOffset": {
+								"l": 51
```

Comment:
![OnPaste 20230316-202917](https://user-images.githubusercontent.com/13815468/225632151-1bababa2-7918-4c3d-8bbc-213edafc222e.jpg)

Oh, I forgot something, this text should be left aligned with the stars.

Reply:
![image](https://user-images.githubusercontent.com/118117643/225640881-05dfe3a6-13af-42fc-a4a9-4e58341d6b09.png)
![image](https://user-images.githubusercontent.com/118117643/225640922-51ae96be-7503-42b8-baaf-dea657eed6e6.png)

This is what they look like now. I updated the text offset on both banners